services/tables/src/main/java/com/linkedin/openhouse/tables/repository/impl/InternalRepositoryUtils.java

```diff
@@ -26,6 +26,7 @@
 public final class InternalRepositoryUtils {
 
   protected static final String POLICIES_KEY = "policies";
+  protected static final String REPLICATION_SETUP_KEY = "replication.enableSetup";
```

Comment:
I believe this is a leakage of implementation details to expose such internal state as table properties, not to mention the table table properties itself can be lost / modified by users directly. We can certainly make this a preserved key to defend against that argument, but I believe this is the wrong step to start to begin with. 

Let's iterate on slack about what is the problem trying to solve here and understand that better.